### PR TITLE
create up to 4 schemas and 2 rego policy locations

### PR DESCRIPTION
**Pull request by djeebus**
also go fmt and validate on PR

### CODE REVIEW

**Comment by polyrain** [comment]
Can we update the docs with the schema env var?

**Review by polyrain — APPROVED** [review]

File: cmd/flags.go [hunk 1]
```diff
@@ -40,10 +40,18 @@ func newStringOpts() DocOpt[string] {
...
 
+func newStringSliceOpts() DocOpt[[]string] {
+	return DocOpt[[]string]{}
+}
+
 func stringFlag(flags *pflag.FlagSet, name, usage string, opts ...DocOpt[string]) {
 	addFlag(name, usage, opts, flags.String, flags.StringP)
 }
 
+func stringSliceFlag(flags *pflag.FlagSet, name, usage string, opts ...DocOpt[[]string]) {
```
Comment (polyrain) [inline cmd/flags.go:51]: Is there a reason `name` is untyped? All the usages are strings, is there a situation where this wouldn't be expected?

**Review by polyrain — APPROVED** [review]
LGTM, minor note but codes pretty easy to follow. Ship it!

File: pkg/validate/validate.go [hunk 2]
```diff
@@ -32,70 +28,44 @@ const kubeconformCommentFormat = `
...
+		// schemas included in kubechecks
+		"default",
 
-func getSchemaLocations() []string {
-	getSchemasOnce.Do(func() {
-		ctx := context.Background()
-		_, span := otel.Tracer("Kubechecks").Start(ctx, "GetSchemaLocations")
-		schemasLocation := viper.GetString("schemas-location")
+		// schemas collected globally
+		"https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/",
```
Comment (davidwin93) [inline pkg/validate/validate.go:38]: should we make this configurable? Maybe via env vars.

**Review by davidwin93 — APPROVED** [review]

File: cmd/flags.go (hunk 1 above)
Comment (djeebus) [reply cmd/flags.go:51]: `name, usage string` is just shorthand for `name string, usage string`.

File: pkg/validate/validate.go (hunk 2 above)
Comment (djeebus) [reply pkg/validate/validate.go]: Sorry, looks like this references the same repo as `"default"`, and doesn't need to be repeated.

**Pull request merged**